packages/test/test-end-to-end-tests/src/test/gc/gcSweepDataStores.spec.ts

```diff
@@ -227,31 +221,15 @@ describeNoCompat("GC data store sweep tests", (getTestObjectProvider) => {
 		}
 
 		// TODO: Receive ops scenarios - loaded before and loaded after (are these just context loading errors?)
-		// TODO: load deleted datastore scenarios
-		itExpects.skip(
+		itExpects(
 			"Requesting swept datastores fails in client loaded after sweep timeout and summarizing container",
 			[
-				// Interactive client's request
-				{
-					eventName: "fluid:telemetry:ContainerRuntime:GC_Tombstone_DataStore_Requested",
-					headers: expectedHeadersLogged.request,
-				},
-				// Interactive client's request w/ allowTombstone
-				{
-					eventName: "fluid:telemetry:ContainerRuntime:GC_Tombstone_DataStore_Requested",
-					category: "generic",
-					headers: expectedHeadersLogged.request_allowTombstone,
-				},
 				{
-					eventName:
-						"fluid:telemetry:ContainerRuntime:GarbageCollector:SweepReadyObject_Loaded",
+					eventName: "fluid:telemetry:ContainerRuntime:GC_Deleted_DataStore_Requested",
 				},
 				// Summarizer client's request
 				{
-					eventName: "fluid:telemetry:ContainerRuntime:GC_Tombstone_DataStore_Requested",
-					category: "generic",
-					headers: expectedHeadersLogged.request,
-					isSummarizerClient: true,
```

Comment:
I can't comment on the unchanged lines. So, adding comments here:
1. Line 271: It should be asserting summarizerResponse.value. It is validating errorResponse twice.
2. You should also validate that "TombstoneResponseHeaderKey" is not set to assert that this error could be coming from because of tombstoning.